build-http.sh

```diff
@@ -15,22 +15,29 @@ declare -r IMAGE_ORIGINAL_TAG="nginx:1.[0-9][0-9]?-alpine"
 
 declare -r IMAGE_TAG="nginx:${VERSION_NGINX}-alpine"
 declare -r USABILLA_TAG_PREFIX="usabillabv/php"
-declare -r USABILLA_TAG="${USABILLA_TAG_PREFIX}:nginx${VERSION_NGINX}"
-declare -r USABILLA_TAG_DEV="${USABILLA_TAG}-dev"
-
-TAG_FILE="./tmp/build-${IMAGE}.tags"
+if [[ ! -v DOCKER_BUILD_PLATFORM ]]; then
+   DOCKER_BUILD_FLAGS=""
+   declare -r USABILLA_TAG_SUFFIX=""
+else
+   DOCKER_BUILD_FLAGS="--platform="${DOCKER_BUILD_PLATFORM}""
+   declare -r USABILLA_TAG_SUFFIX="-$(echo "${DOCKER_BUILD_PLATFORM}" | cut -d "/" -f 2)"
```

Comment:
If we are ignoring the first part of `DOCKER_BUILD_PLATFORM` in the tag, we may as well expose only the architecture as a parameter, otherwise if someone tries to add `DOCKER_BUILD_PLATFORM=windows/amd64` later it will push to the same tag.

Reply:
Let me rework this to include the OS part as well 👍